backend/api/game.go

```diff
@@ -43,20 +42,24 @@ func getGamesWithTags(c echo.Context) error {
 		log.WithError(err).Error("Database connection error in API getGamesWithTags")
 		return c.JSON(http.StatusInternalServerError, "Database connection error")
 	}
+	// // grab tags from context
+	// rawTags := c.FormValue(TAGS)
 
-	// grab tags from context
-	rawTags := c.Param("tags")
+	// // split string into array
+	// tags := strings.Split(rawTags, ",")
 
-	// split string into array
-	tags := strings.Split(rawTags, ",")
+	// // fetch games with tags
+	tags := c.QueryParam("tags")
+	tagsArray:= strings.Split(tags, "-")
+	games, err := _db.GetGamesByTags([]string{tagsArray[0], tagsArray[1]}, false)
```

Comment:
This limits the tags search to two tags. Please don't change code unless you are assigned to it.